.github/workflows/snyk.yml

```diff
@@ -13,6 +13,7 @@ jobs:
       ORG: guardian-dotcom-n2y
       SKIP_NODE: false
       SKIP_GO: false
+      SKIP_SBT: true
```

Comment:
To get rid of error

```
Error: No file in /home/runner/work/automat/automat matched to [**/*.sbt,**/project/build.properties,**/project/**.{scala,sbt}], make sure you have checked out the target repository
```

Reply:
Triggered the job manually from this branch and it was [successful](https://github.com/guardian/automat/actions/runs/4345787436/jobs/7590979115).